src/material/_theming.scss

```diff
@@ -0,0 +1,8 @@
+// Forwards all public API mixins so they can be imported from a single entry point.
+// Note that we have to forward the `.import` files for backwards-compatibility with
+// projects that don't use Sass modules and include the `mat-`-prefixed mixins.
+
+@forward './core/color/all-color.import';
+@forward './core/density/private/all-density.import';
+@forward './core/theming/all-theme.import';
+@forward './core/typography/all-typography.import';
```

Comment:
Potentially for a follow up: it would be nice to have some sort of public api gold for Sass, since it's hard to tell from here all the stuff this is transitively forwarding.

Reply:
I suspect that this will be a non-trivial amount of work since the PostCSS AST which we're using for Sass doesn't have the same APIs as the ones we use to generate the TS goldens. The Sass compiler probably has a better AST, but I don't know if it's public.